x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/table_row_actions.tsx

```diff
@@ -113,6 +116,26 @@ export const TableRowActions: React.FunctionComponent<{
         />
       </EuiContextMenuItem>
     );
+
+    if (agentTamperProtectionEnabled) {
+      menuItems.push(
+        <EuiContextMenuItem
+          icon="minusInCircle"
+          onClick={() => {
+            onGetUninstallCommandClick();
+            setIsMenuOpen(false);
+          }}
+          disabled={!agent.active}
+          key="getUninstallCommand"
+          data-test-subj="uninstallAgentMenuItem"
+        >
+          <FormattedMessage
+            id="xpack.fleet.agentList.getUninstallCommand"
+            defaultMessage="Uninstall agent"
```

Comment:
nit: The action name is a little misleading, it could be called "get uninstall command". Maybe this was already discussed with product, in case it's fine.

Reply:
good point, it was changed from 'Get uninstall command' to 'Uninstall agent' by product, as it is similar to 'Add agent' which also just opens a flyout showing commands to the user